lib/galaxy/tools/deps/views.py

```diff
@@ -125,5 +126,8 @@ def installable_resolvers(self):
         """
         return [index for index, resolver in enumerate(self._dependency_resolvers) if hasattr(resolver, "install_dependency") and not resolver.disabled ]
 
-    def get_requirements_status(self, requested_requirements):
-        return [self.manager_dependency(**req) for req in requested_requirements]
+    def get_requirements_status(self, requested_requirements, installed_tool_dependencies=None):
+        dependencies = deepcopy(requested_requirements)  # Make a copy to avoid changing contents of requested_requirements
```

Comment:
This should be a better fix: https://gist.github.com/nsoranzo/02a301a12ebf8a650ea8e1175ede0444

I can commit it to your branch if you want.